lib/adapter.js

```diff
@@ -1117,8 +1106,9 @@ module.exports = (function() {
    * @param  {Function} fn
    * @param  {[type]}   cb
    */
-  function spawnConnection(connectionName, fn, cb) {
+  function spawnConnection(queryType, connectionName, fn, cb) {
```

Comment:
Should be careful that this doesn't break anywhere. Did you check for all uses? Does this get exposed to user code? (if so, it's a BC break).